src/deb/tomcat7/conf/server.template.xml

```diff
@@ -19,7 +19,7 @@
      define subcomponents such as "Valves" at this level.
      Documentation at /docs/config/server.html
  -->
-<Server port="${CONTROL_PORT}" shutdown="SHUTDOWN">
+<Server port="-1" shutdown="SHUTDOWN">
```

Comment:
Is this a convention used elsewhere, or should it be?

Reply:
I think we should use it for all the java stacks if it works

Reply:
I agree. I'd like to do this in the plos match states if it works . . . does it work?

Reply:
It seems to...